generated/api.github.com.json

```diff
@@ -992,7 +992,7 @@
                 "properties": {
                   "access_token": {
                     "type": "string",
-                    "description": "**Required.** The OAuth access token used to authenticate to the GitHub API.",
+                    "description": "The OAuth access token used to authenticate to the GitHub API.",
```

Comment:
I don't think this change is correct, and it already seems to have been reverted, it says required in the docs https://docs.github.com/en/rest/reference/apps#create-a-scoped-access-token--parameters

<img width="765" alt="image" src="https://user-images.githubusercontent.com/39992/112734824-678f8700-8f05-11eb-89e8-2a3db5e53817.png">

Reply:
ah nevermind, the property is required via the JSON schema